src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -502,14 +502,19 @@ public static String injectContextDependentValuesForTest(String content) {
     }
 
     private static TimeZone getTimeZone(String content) {
+        //Searches for last String of pattern "UTC+0800" in the content.
         Pattern pattern = Pattern.compile(REGEX_TIMEZONE_OFFSET);
         Matcher matcher = pattern.matcher(content);
-        //set default timezone offset.
+        //set default time zone offset.
         String timeZoneOffset = "+0000";
         while (matcher.find()) {
```

Comment:
because using pattern `+xxxx` it also matches string such as responseCommentRow-`1234`567812345678 in the content file.